ui/components/ui/chip/README.mdx

```diff
@@ -0,0 +1,15 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
```

Comment:
Is there anyway to prevent this from showing in the markdown?

Reply:
I'm not sure. But I know what you mean. Reading the README.mdx outside of Storybook in a code editor or github isn't the best developer experience